### DISCUSSION TITLE
Name of the repository and/or package (analysis)

### DISCUSSION THREAD

**Discussion by nbortolotti** [nbortolotti/tflitego#10] [Q&A]
I started the initial version with tflitego, but I receives interesting feedback related to the limitation to use "go" as part of the repository name by @marcoshuck . (currently equivalent to the package name)

 Any thoughts on that? examples, ideas, recommendations?

**Comment by nbortolotti** [comment]
Adding more context to evaluate the case. https://blog.golang.org/package-names

**Reply by marcoshuck** [reply]
There's a library for using JWT that could help us deciding what to do:

The repository name is `jwt-go` but when you import it, the package name is `jwt`

https://github.com/dgrijalva/jwt-go

**Reply by nbortolotti** [reply]
Let's do that, I will maintain the name of the repository and we will prepare a PR with the package modification to tflite. Thanks @marcoshuck
